### DISCUSSION TITLE
be curious???

### DISCUSSION THREAD

**Discussion by lixcxc96** [swar/Swar-Chia-Plot-Manager#1088] [General]
Dear: Swar Swar Patel

First of all, thank you very much for your selfless dedication,

I see this software can achieve a 2T, a 1t solid-state disk, 64GB of memory, CPU unknown, one day can p figure 3.2tb, how do you do it, I want to know! I sincerely look forward to your sharing. thank!
![微信图片_20210602214438](https://user-images.githubusercontent.com/84433255/120492308-8a01bd00-c3ec-11eb-86b2-29dd64870e29.png)

**Comment by lixcxc96** [comment]
cpu Intel  I7 10700K
ram 32gb ddr4 3200 
2x 2TB samsung evo plus  nvme
1x 16TB hhd in computer
![微信图片_20210603035624](https://user-images.githubusercontent.com/84433255/120544505-12e61c00-c420-11eb-9758-57a8343d750c.png)
------------------------------------------------------------------------------------------------
my config:
  max_concurrent: 16
  max_for_phase_1: 7
  minimum_minutes_between_jobs: 1

jobs:
  - name: 2T_disk1
    max_plots: 999
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: G:\T1
    temporary2_directory:
    destination_directory: E:\
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 4
    max_concurrent_with_start_early: 8
    initial_delay_minutes: 2
    stagger_minutes: 0
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 256
    enable_cpu_affinity: true
    cpu_affinity: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 ]

  - name: 2T_disk2
    max_plots: 999
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: H:\T1
    temporary2_directory:
    destination_directory: E:\
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 3
    max_concurrent_with_start_early: 8
    initial_delay_minutes: 2
    stagger_minutes: 0
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 256
    enable_cpu_affinity: true
    cpu_affinity: [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15 ]
---------------------------------------------------------------------------------------------------------
Why can't I do it? Is it better to have a paid version?

**Reply by gitrid** [reply]
What is your daily output?

**Reply by lixcxc96** [reply]
2TB
